app/services/personal_key_formatter.rb

```diff
@@ -1,11 +1,15 @@
 class PersonalKeyFormatter
+  CHAR_COUNT = RandomPhrase::WORD_LENGTH
+  WORD_COUNT = Figaro.env.recovery_code_length.to_i
+  VALID_CHAR = '[a-zA-Z0-9]'.freeze
+  VALID_WORD = "#{VALID_CHAR}{#{CHAR_COUNT}}"
+  REGEXP = "(?:#{VALID_WORD}([\s-])?){#{WORD_COUNT - 1}}#{VALID_WORD}".freeze
+
   def regexp
```

Comment:
yeah totally fine with that, I wasn't sure why it was being called with `.new` in the first place 😄